### PR TITLE
Remove unused script options

### PR DESCRIPTION
**Pull request by richardzilincikPantheon**
Remove script options which are currently
unused and unlikely to be used in the future.
Pull the paths to ietf drafts and rfcs from
the `ietf-drafts` and `ietf-rfcs` config
options respectively.

### CODE REVIEW

**Comment by SlavomirMazurPantheon** [comment]
RFC path now does not contain `/` character at the end which might produce some errors:
```
Traceback (most recent call last):
  File "/./sdo_analysis/bin/yangIetf.py", line 217, in <module>
    draftExtractor.extract_drafts()
  File "/sdo_analysis/bin/extractors/dratfExtractor.py", line 94, in extract_drafts
    shutil.copy2(draft_file_path, self.draft_path_strict)
  File "/usr/lib/python3.6/shutil.py", line 263, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/var/yang/ietf/my-id-archive-mirrordraft-abad-i2nsf-sdn-ipsec-flow-protection-02.txt'
```

File: bin/generatePrivatePage.py [hunk 1]
```diff
@@ -94,7 +89,7 @@ def main():
         os_dir = os.path.join(huawei_dir, huawei_os)
         specific_os_dir = [name for name in os.listdir(os_dir) if os.path.isdir(os.path.join(os_dir, name))]
         for directory_specific_os in specific_os_dir:
-            context['huawei'].append({'alphaNumeric': re.sub(r'\W+', '', '{}{}'.format(huawei_os, directory_specific_os)),
+            context['huawei'].append({'alphaNumeric': re.sub(r'\W+', '', os.path.join(huawei_os, directory_specific_os)),
```
Comment (SlavomirMazurPantheon) [inline bin/generatePrivatePage.py:92]: I think that we are not composing path here - only string

File: bin/yangIetf.py [hunk 2]
```diff
@@ -152,11 +145,12 @@ def custom_print(message: str):
                         help='Optional flag that determines wheter compilation should be run '
                              'for all files even if they have not been changed '
                              'or even if the validators versions have not been changed.',
-                        type=bool,
-                        default=False)
+                        action='store_true')
```
Comment (SlavomirMazurPantheon) [inline bin/yangIetf.py:148]: `default=False` should stay as original

File: bin/yangIetf.py [hunk 3]
```diff
@@ -72,74 +71,68 @@ def custom_print(message: str):
...
     parser = argparse.ArgumentParser(description='YANG RFC/Draft Processor')
-    parser.add_argument('--draftpath',
-                        help='Path to the directory where all the drafts will be stored. '
-                             'Default is {}/my-id-mirror/ '
-                             'To get expired drafts as well, use {}/my-id-archive-mirror/'
-                             .format(ietf_directory, ietf_directory),
-                        type=str,
+    parser.add_argument('--archived',
+                        help='Extract expired drafts as well',
+                        action='store_true',
```
Comment (SlavomirMazurPantheon) [inline bin/yangIetf.py:81]: `default=False` should stay as original

File: bin/generatePrivatePage.py [hunk 4]
```diff
@@ -105,8 +100,8 @@ def main():
         specific_os_dir = [name for name in os.listdir(os_dir) if os.path.isdir(os.path.join(os_dir, name))]
         for directory_specific_os in specific_os_dir:
             context['fujitsu'].append(
-                {'alphaNumeric': re.sub(r'\W+', '', '{}{}'.format(fujitsu_os, directory_specific_os)),
-                 'allCharacters': '{}{}'.format(fujitsu_os, directory_specific_os)})
+                {'alphaNumeric': re.sub(r'\W+', '', os.path.join(fujitsu_os, directory_specific_os)),
+                 'allCharacters': os.path.join(fujitsu_os, directory_specific_os)})
```
Comment (SlavomirMazurPantheon) [inline bin/generatePrivatePage.py:104]: same here

File: bin/runIETFstats.sh [hunk 5]
```diff
@@ -108,10 +108,9 @@ ln -f -s $NONIETFDIR/yangmodels/yang/standard/iana/ $MODULES/iana
 date +"%c: Starting to extract all YANG modules from IETF documents" >>$LOG
 # Using --draftpath $IETFDIR/my-id-archive-mirror/ means much longer process as all expired drafts will also be analyzed...
 if [ $(date +%u) -eq 6 ]; then
-	python $BIN/yangIetf.py --draftpath $IETFDIR/my-id-archive-mirror/ >>$LOG 2>&1
+	python $BIN/yangIetf.py --archived >>$LOG 2>&1
```
Comment (SlavomirMazurPantheon) [inline bin/runIETFstats.sh:111]: FYI: this is actually very good and useful change

File: bin/generatePrivatePage.py (hunk 1 above)
Comment (richardzilincikPantheon) [reply bin/generatePrivatePage.py:92]: Oh yeah, you're right

File: bin/yangIetf.py (hunk 2 above)
Comment (richardzilincikPantheon) [reply bin/yangIetf.py:148]: `default=False` is implicit in `action='store_true'`, but I can add it back to make the formatting more similar to other options.

Comment (SlavomirMazurPantheon) [reply bin/yangIetf.py]: Ah Okay, I thought the default was True, because yangIetf.py started parsing all - even archived drafts for some reason.

Comment (richardzilincikPantheon) [reply bin/yangIetf.py]: That was because I accidentally overrode the default with `default='{}/my-id-mirror/'.format(ietf_directory)`. Fixed now.

**Review by SlavomirMazurPantheon — APPROVED** [review]

**Pull request merged**